webtau-docs/znai/GraphQL/queries-and-mutations.md

```diff
@@ -1,11 +1,11 @@
-# Executing Queries
+# Executing Queries and Mutations
```

Comment:
Actually should we have two sections here. One for queries and one for mutations?

Reply:
I don't think so, there's no difference in how you execute them or how you'd assert